auth/ante.go

```diff
@@ -100,7 +83,7 @@ func NewAnteHandler(
 
 		// gas for tx
 		gasForTx := gasWantedPerTx // stdTx.Fee.Gas
-		feeForTx := feeWantedPerTx // stdTx.Fee.Amount
+		feeForTx := FeeWantedPerTx // stdTx.Fee.Amount
```

Comment:
Why create this variable at all?

Reply:
Just to create logical block, nothing else.

Reply:
Cool